modules/article-page/views/show.html

```diff
@@ -5,13 +5,23 @@
 {% block sideNav %}{% endblock %}
 
 {% block main %}
-  <section class="blurb">{% area article, 'blurb' %}</section>
+  <div class="article-show">
+    <h4 class="meta">{{ data.page.title}}</h4>
+    <h1>{{ data.piece.title }}</h1>
+    <section class="blurb">
```

Comment:
Right now `blurb` is wrapping the article body. There's a good case here for a `header` wrapping the "breadcrumb," title, and blurb.